packages/website/src/pages/foundations/Iconography.tsx

```diff
@@ -1,34 +1,102 @@
-import {svg as Icons, SvgName} from '@coveord/plasma-style';
+import {ITableHOCProps, TableHOC, tableWithFilter, TableWithFilterProps} from '@coveord/plasma-react';
+import * as PlasmaReactIcons from '@coveord/plasma-react-icons';
+import classNames from 'classnames';
+import dynamic from 'next/dynamic';
 import * as React from 'react';
-import {LinkSvg} from '@coveord/plasma-react';
 
-import PlasmaComponent from '../../building-blocs/PlasmaComponent';
+import {PageHeader} from '../../building-blocs/PageHeader';
+import {PlasmaLoading} from '../../building-blocs/PlasmaLoading';
 
-export const IconographyExamples = () => (
-    <PlasmaComponent
-        id="IconsList"
-        title="Icons list"
-        usage={<p>Icons are used to visually represent actions, functionalities, and features.</p>}
-    >
-        <ul className="sg-icons-list">
-            {Object.keys(Icons)
-                .sort()
-                .map((svgName: SvgName) => (
-                    <IconItem key={svgName} svgName={svgName} fileName={Icons[svgName].fileName} />
-                ))}
-        </ul>
-    </PlasmaComponent>
+const {iconsList, ...Icons} = PlasmaReactIcons;
+
+const Sandbox = dynamic(
+    import('../../building-blocs/Sandbox').then((mod) => mod.Sandbox),
+    {ssr: false, loading: () => <PlasmaLoading />}

```

Comment:
I really like the built-in loading prop in Next :eyes: 
So much cleaner that the if(loading) {return <Loading />} else { return <MyPage />}